data/other/timezones/ko.yml

```diff
@@ -1,139 +1,139 @@
 ---
 ko:
   timezones:
-    Etc/GMT+12: "(GMT-12 : 00) 국제 날짜 변경선 West"
```

Comment:
We should be consistent about spacing here.  Notice that, e.g., line 4 has no space around the `-` in `GMT-12`, but line 38 has spaces around the `+`.

Reply:
I believe this is resolved now